src/validator/validator.ts

```diff
@@ -1,26 +1,31 @@
+import * as airnodeValidator from '@api3/airnode-validator';
+import * as airnodeOis from '@api3/ois';
 import { z, ZodFirstPartySchemaTypes } from 'zod';
 
 export const templateParametersSchema = z.object({ type: z.string(), name: z.string(), value: z.string() });
 
-export const triggerSchema = z.object({
+export const rrpBeaconServerKeeperJobsTriggerSchema = z.object({
   chainIds: z.array(z.string()),
-  templateId: z.string(),
+  templateId: z.string().regex(/^0x[a-fA-F0-9]{64}$/),
```

Comment:
Airnode exports an `evmIdSchema` that could work here?

Reply:
Yes! and I can probably also use the `evmAddressSchema` from airnode-validator